gleam.toml

```diff
@@ -11,9 +11,9 @@ links = [
 gleam = ">= 0.33.0"
 
 [dependencies]
-gleam_stdlib = "~> 0.34"
-birl = "~> 1.0"
+gleam_stdlib = ">= 0.34.0 and < 2.0.0"
+birl = ">= 1.7.0 and < 2.0.0"
+gleam_json = "> 1.0.0 and < 2.0.0"
```

Comment:
I think this dep has been moved by mistake?

Reply:
Oops! Good catch